### ISSUE TITLE
Wildcard Cert Registered under GoDaddy - Not Self Signed.

### ISSUE THREAD

**Issue by cengbrecht** [aykit/MyOwnNotes#109]
I paid for a 700$ cert from Godaddy to make everything I have work. Now yours is the only one that does not.
https://owncloud.brokedcomputer.com

**Comment by jalcine** [comment]
Side-note: GoDaddy's a bad place for SSL certificates (WTF $700 USD!).

Have you tried importing the certificate that you'd export from your Web browser to your local computer, copying that to your phone and then importing that?

**Comment by cengbrecht** [comment]
The whole point was that I shouldn't need too.

**Comment by jalcine** [comment]
Possibly. There's a chance that the certificate chain you have might not exist on the version of Android you're using. _A chance_. This is Android we're talking about.

**Comment by cengbrecht** [comment]
Hahahaha, this is true too... Though thats kind of annoying. Also, where do you get professional SSL's from that GoDaddy is the worst? Also it was for 2 years, 700$ for a Wildcard cert is pretty good.

**Comment by ramschmaerchen** [comment]
@cengbrecht could you do me a favor and enable "extensive logcat output" in settings and post your logcat output here?

Btw:
http://cheapsslsecurity.com
Buying an expensive certificate does not make it more secure.

**Comment by jalcine** [comment]
It doesn't make it more secure, but with the (silly) way that CAs are setup, you have to pay more for more implicit trust from the browser.

**Comment by cengbrecht** [comment]
Which I paid for, but then found out that Android was stupid at times. :P 

Also, I tried adding the cert manually, and it didn't take. I did follow the instructions. I've had this issue in the past though, I think its my version of Android, I am running Ressurrection Remix. http://forum.xda-developers.com/google-nexus-5/development/rom-resurrection-remix-4-4-2-kitkat-v5-t2706610

So that may play into it. However, are you sure that the application is setup for *.certs? (Wildcard)
